src/core/memory.cpp

```diff
@@ -51,3 +51,25 @@ void Memory::free_static(void *p_ptr) {
 void *operator new(size_t p_size, const char *p_description) {
 	return godot::Memory::alloc_static(p_size);
 }
+
+using namespace godot;
+
+#ifdef _MSC_VER
+void operator delete(void *p_mem, const char *p_description) {
+
```

Comment:
Just a style nitpick: it should not have an empty line between the function signature and the body. Same for the others down below.

Reply:
that's what I get for copy pasting from Godot source:) I'll fix it up :)